bittide/src/Bittide/Axi4.hs

```diff
@@ -0,0 +1,108 @@
+-- SPDX-FileCopyrightText: 2023 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE FlexibleContexts #-}
+
+{-# OPTIONS_GHC -fconstraint-solver-iterations=10 #-}
+
+module Bittide.Axi4 where
+
+import Clash.Prelude
+
+import Protocols
+import Protocols.Axi4.Stream
+
+import Bittide.Extra.Maybe
+
+
+{-# NOINLINE axisFromByteStream #-}
+-- | Transforms an Axi4 stream of 1 byte wide into an Axi4 stream of n bytes wide.
+-- It uses an internal state machine to replace the bytes in the outgoing stream one by one.
+-- If it encounters `_tlast` or has captured enough bytes, it will present the transaction
```

Comment:
```suggestion
-- If it encounters '_tlast' or has captured enough bytes, it will present the transaction
```

Backticks aren't picked up by Haddock AFAIK. But also, these are actual fields so Haddock should be able to link them if you use `'`.